kokoro/presubmit.sh

```diff
@@ -11,7 +11,7 @@ export RUST_BACKTRACE=1
 export RUST_LOG=debug
 
 ./scripts/docker_pull
-./scripts/docker_run cargo nextest run --hide-progress-bar
+./scripts/docker_run cargo nextest run --all-targets --hide-progress-bar
```

Comment:
I'm assuming this is what results in benches being run in CI? Seems suprising.

Reply:
Yeah I was confused too, I always assumed --all-targets means it tries all the possible --targets, instead it's really so it also runs benches (with a single iteration, AFAICT) and examples, which is what we want. Added a comment.

Reply:
Sweet, thx for adding the comment!